src/OpenTelemetry.Instrumentation.StackExchangeRedis/StackExchangeRedisCallsInstrumentationOptions.cs

```diff
@@ -21,5 +22,9 @@ namespace OpenTelemetry.Instrumentation.StackExchangeRedis
     /// </summary>
     public class StackExchangeRedisCallsInstrumentationOptions
     {
+        /// <summary>
+        /// Gets or sets the maximum time that should elapse between flushing the internal buffer to the configured Jaeger agent. Default value: 00:00:10.
```

Comment:
Not sure what this comment about Jaeger means?

Reply:
Sorry copy and paste fail. How about: `Gets or sets the maximum time that should elapse between flushing the internal buffer of Redis profiling sessions and creating <see cref="Activity"/> objects. Default value: 00:00:10.`?

Reply:
just updated

Reply:
@eddynaka Would you mind making that change for me since you are working on the other feedback?

Reply:
just updated! :)